apps/web/src/shared/ui/inputs/multi-select-search-input.ui.tsx

```
@@ -0,0 +1,130 @@
+  return (
+    <InputField>
+      {label && (
+        <InputLabel>
+          {label} {required && <span className="text-tint-red">*</span>}
+        </InputLabel>
+      )}
+      <Combobox open={open} onOpenChange={setOpen}>
+        <ComboboxTrigger disabled={disabled} className="font-normal">
+          {!!value && value.length > 0 ?
+            value.map((v) => (
+              <Tag variant="outline" size="small">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)